src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -399,4 +404,12 @@ private function escapeHtml($str)
     {
         return htmlspecialchars($str, ENT_COMPAT | ENT_SUBSTITUTE, $this->charset);
     }
+    
+    /**
+     * Returns link to search query on Google
+     */
+    private function formatGoogleLink($query)
+    {
+        return '<a href="https://google.com/search?q=' . urlencode(strip_tags($query)) . '" target="_blank" rel="noopener noreferrer" title="Search error on Google">Search on Google</a>';
```

Comment:
@stof my 2 cent, probably the title in "Search error on Google" and text link "Search on Google" should support l18n, using hardcoded english words probably is not a good idea.